lib/utils.sh

```diff
@@ -152,6 +152,9 @@ resolve_version() {
 
   if [ "$query" = lts ] || [ "$query" = "lts/*" ]; then
     query="${nodejs_codenames[${#nodejs_codenames[@]} - 1]#*:}"
+    local all_versions
+    all_versions=$("$ASDF_NODEJS_PLUGIN_DIR/bin/list-all" 2>/dev/null | tr ' ' '\n')
+    query=$(echo "$all_versions" | grep "^$query\." | tail -n1)
```

Comment:
Why did this need to be added?

Reply:
I see, pulling in a fix from another PR. Probably best not to include multiple fixes in the same PR, it's already hard enough to get a PR accepted, and the one containing this fix is still open. I'm going to wait on @augustobmoura to review these 3 lines.